package.json

```diff
@@ -14,25 +14,23 @@
     ]
   },
   "scripts": {
-    "build": "yarn lerna run build",
-    "clean": "yarn lerna run clean",
-    "lint": "yarn lerna run lint --stream",
-    "test": "yarn lerna run test --stream --concurrency 1",
-    "browserify": "yarn lerna run browserify",
-    "bundlewatch": "yarn lerna exec yarn run bundlewatch",
```

Comment:
These are specific to `near-api-js` so i've removed them from the workspace root